Translated description in english (#13)
neopixl/tech-radar: radar/2023-06-10/Alamofire.md

---
title: "Alamofire"
ring: assess
quadrant: iOS
tags: [iOS]
---

<p>Network calls framework</p>
<p><a href="https://github.com/Alamofire/Alamofire">Link to Github</a> <br /> <br />

> Can still be used in old projects.<br />
> However, it would be better to use the native language to make network calls: <a href="https://developer.apple.com/documentation/foundation/urlrequest">Apple doc</a></p>

Reviewer: I suggest we use our convictions in the tech-radar. Thus, we might express our position in a stronger way, e.g.

```suggestion
Can still be found in legacy projects.<br />
However, we strongly recommend to use the native language to make network calls: <a href="https://developer.apple.com/documentation/foundation/urlrequest">Apple doc</a></p>
```

Author: Since it's an iOS tech I'll let @NeopixlAnthony (or someone else) complete this part
